server/character.lua

```diff
@@ -6,10 +6,53 @@ local function getAllowedAmountOfCharacters(license2, license)
     return QBCore.Config.Characters.PlayersNumberOfCharacters[license2] or license and QBCore.Config.Characters.PlayersNumberOfCharacters[license] or QBCore.Config.Characters.DefaultNumberOfCharacters
 end
 
+---@param sexString number | string 
+---@return string
+local function getSexString(sexString)
+    if sexString ~= 1 then
+        sexString = 'M'
+    else
+        sexString = 'F'
+    end
+    return sexString
+end
+
 ---@param source Source
 local function giveStarterItems(source)
-    for _, v in pairs(QBConfig.StarterItems) do
-        exports.ox_inventory:AddItem(source, v.item, v.amount)
+    local Player = QBCore.Functions.GetPlayer(source)
+
+    for _, v in pairs(Config.StarterItems) do
+        if v.type == 'id' then
+            local metadata = {
+                cardtype = v,
+                citizenid = Player.PlayerData.citizenid,
+                firstname = Player.PlayerData.charinfo.firstname,
+                lastname = Player.PlayerData.charinfo.lastname,
+                birthdate = Player.PlayerData.charinfo.birthdate,
+                sex =  getSexString(Player.PlayerData.charinfo.gender),
+                nationality = Player.PlayerData.charinfo.nationality,
```

Comment:
metadata should be in the config itself, even if that means putting a function in the config